week03_model_free/homework.ipynb

```diff
@@ -766,6 +766,49 @@
    },
    "outputs": [],
    "source": []
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "%%writefile ../run_display_servers.sh\n",
```

Comment:
This file should be kept separately instead of being created from within Jupyter.

Reply:
Ok